protocol/metaModel.json

```diff
@@ -5346,7 +5346,7 @@
 						}
 					},
 					"optional": true,
-					"documentation": "Marks that the code action cannot currently be applied.\nClients should follow the following guidelines regarding disabled code actions:\n  - Disabled code actions are not shown in automatic [lightbulb](https://code.visualstudio.com/docs/editor/editingevolved#_code-action)\n    code action menu.\n  - Disabled actions are shown as faded out in the code action menu when the user request a more specific type\n    of code action, such as refactorings.\n  - If the user has a [keybinding](https://code.visualstudio.com/docs/editor/refactoring#_keybindings-for-code-actions)\n    that auto applies a code action and only a disabled code actions are returned, the client should show the user an\n    error message with `reason` in the editor.\n@since 3.16.0",
```

Comment:
I think the linebreak removal should be more selective. In this case, there are list items that got mashed together:

```
  /// Marks that the code action cannot currently be applied.
  ///
  /// Clients should follow the following guidelines regarding disabled code
  /// actions:
  ///
  /// - Disabled code actions are not shown in automatic lightbulbs code
  ///   action menus.
  ///
  /// - Disabled actions are shown as faded out in the code action menu when
  ///   the user request a more specific type of code action, such as
  ///   refactorings.
```

Ideally, the linebreaks before the list items would not be removed, but those that are only wrapping text would. This might be tricky if the source of truth requires the word-wrapping, though replacing out only `\b\n\b` might get most of the way (so any line ending with a fullstop would not have its newline stripped).